.github/workflows/main.yaml

```diff
@@ -0,0 +1,87 @@
+name: main
+
+on:
+  push:
+    branches:
+      - "master"
+    tags:
+      - "v*"
+
+jobs:
+  quality:
+    runs-on: ubuntu-latest
+    steps:
+      - name: compute checksums of auto generated files
+        run: |
+          echo hello world
+
+      - name: lint
+        run: | 
+          make test
```

Comment:
lint instead of test?

Reply:
yes, i temporarily set this to test, as some linting errors are still alive

Reply:
Fixed